.github/workflows/swift_matrix.yml

```diff
@@ -168,6 +169,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           persist-credentials: false
+          format_check_enabled: true
```

Comment:
Nit but do we want to do this in a separate PR, since it's not related to checking out submodules?

Reply:
Yikes, didn't mean to do that.  Will take ti out.